lib/find-all-matching-brackets.js

```diff
@@ -10,12 +11,14 @@ module.exports = function(str, ignoreMismatches) {
   for (let i = 0; i < str.length; i++) {
     const current = stack.length > 0 ? stack[stack.length - 1] : null;
     const currentKind = current ? current.kind : '';
+    // eslint-disable-next-line default-case
```

Comment:
@addaleax is this safe / intentional?

Reply:
That `default:` is missing? Yeah. You can just add `default: break;` though, if you think that would be preferable to just silencing the linter warning